formats/help/publisher/examples/help_page.json

```diff
@@ -0,0 +1,32 @@
+  },
+  "details": {
+    "body": "$!GOV.UK is the website for the UK government. It's the best place to find government services and information.$!\r\n\r\nThe site is maintained by the [Government Digital Service (GDS).](http://digital.cabinetoffice.gov.uk/)\r\n\r\n^GOV.UK has replaced Business Link and Directgov.^\r\n\r\n##What’s on GOV.UK\r\n\r\nYou can find: \r\n\r\n- information and services for citizens and businesses\r\n- detailed guidance for professionals\r\n- information on government and policy\r\n\r\n##How GOV.UK is updated\r\n\r\nWe work closely with every government department to make sure all the information on GOV.UK is accurate and up to date. \r\n\r\nGOV.UK is also updated based on results from user testing and feedback from members of the public.\r\n\r\n##Tracking users\r\n\r\nWe track how you use GOV.UK, but we don't collect or store your personal information (eg your name or address) while you're browsing. This means that you can't be personally identified.\r\n\r\nWe use Google Analytics software to track:\r\n\r\n- what pages you visit\r\n- how long you use the site\r\n- how you got to the site and what links you clicked on \r\n\r\nWe don't allow Google to share this data with anyone else. \r\n\r\n##Reuse information published on GOV.UK\r\n\r\nYou can republish any of the content on GOV.UK, as long as you meet the conditions of the [Open Government Licence.](http://www.nationalarchives.gov.uk/doc/open-government-licence/) \r\n\r\n##Links to other sites from GOV.UK\r\n\r\nGOV.UK links to external organisations when that link is essential for helping someone complete a task that they begin on the site. \r\n\r\nFor example, there’s a link from the content on university applications to the Universities and Colleges Admissions Service (UCAS) website.\r\n\r\nWe will only consider link requests if the website falls into this ‘essential’ category. \r\n\r\nYou can [request a link to be added](/feedback/contact) to GOV.UK. \r\n\r\n##Data sets on GOV.UK\r\n\r\nGOV.UK uses several data sets to help make the website better. \r\n\r\nFor example, data is used to take you to your council's website if you want to know what day your bin's collected.\r\n\r\nMany of these data sets are used under the [Open Government Licence.](http://www.nationalarchives.gov.uk/doc/open-government-licence/)\r\n\r\nWe use:\r\n\r\n- [Local Government Service List](http://id.esd.org.uk/ServiceList) by [esd-toolkit](http://esd.org.uk/)\r\n- [Code-Point Open](http://www.ordnancesurvey.co.uk/oswebsite/products/code-point-open/index.html) and [Boundary-Line](http://www.ordnancesurvey.co.uk/oswebsite/products/boundary-line/index.html) from [Ordnance Survey](http://www.ordnancesurvey.co.uk/oswebsite/)\r\n- [ONS Postcode Directory](http://www.ons.gov.uk/ons/guide-method/geography/products/postcode-directories/-nspp-/index.html) from [Office for National Statistics](http://www.ons.gov.uk/ons/index.html)\r\n\r\n\r\n*[GDS]: Government Digital Service"
```

Comment:
We've updated the Publisher example to show that we're now pushing govspeak, but correctly annotated for publishing-api to render.